src/cosl/coordinated_workers/coordinator.py

```diff
@@ -63,6 +64,8 @@
 
 logger = logging.getLogger(__name__)
 
+S3_CA_CERT_PATH = "./s3_ca.crt"
```

Comment:
maybe its better to keep the path uniform with other certs? (e.g: /etc/worker/s3_ca.crt)

Reply:
yeah I was still thinking in terms of charm container